enterprise/internal/codeintel/gitserver/client.go

```diff
@@ -426,3 +437,13 @@ func (c *Client) repositoryIDToRepo(ctx context.Context, repositoryID int) (api.
 
 	return api.RepoName(repoName), nil
 }
+
+func isRevisionNotFound(err error) bool {
+	for ex := err; ex != nil; ex = errors.Unwrap(ex) {
+		if gitserver.IsRevisionNotFound(ex) {
+			return true
+		}
+	}
+
+	return false
+}
```

Comment:
```suggestion
func isRevisionNotFound(err error) bool {
	return errors.Is(err, &gitserver.RevisionNotFoundError{})
}
```

Or just inline that.